src/main/java/fi/otavanopisto/kuntaapi/server/integrations/management/ManagementMenuProvider.java

```diff
@@ -111,99 +108,97 @@ public MenuItem findOrganizationMenuItem(OrganizationId organizationId, MenuId m
       return null;
     }
     
-    MenuId mwpMenuId = idController.translateMenuId(menuId, MwpConsts.IDENTIFIER_NAME);
-    if (mwpMenuId == null) {
+    MenuId managementMenuId = idController.translateMenuId(menuId, MwpConsts.IDENTIFIER_NAME);
+    if (managementMenuId == null) {
       logger.severe(String.format(COULD_NOT_TRANSLATE_MENU_ID, menuId.toString()));
       return null;
     }
     
-    MenuItemId mwpItemId = idController.translateMenuItemId(menuItemId, MwpConsts.IDENTIFIER_NAME);
-    if (mwpItemId == null) {
-      logger.severe(String.format("Could not translate menu item id %s to MWP id", menuItemId.toString()));
+    MenuItemId managementItemId = idController.translateMenuItemId(menuItemId, MwpConsts.IDENTIFIER_NAME);
+    if (managementItemId == null) {
+      logger.severe(String.format("Could not translate menu item id %s to management id", menuItemId.toString()));
       return null;
     }
     
-    return findMenuItem(organizationId, mwpMenuId, mwpItemId);
+    return findMenuItem(organizationId, managementMenuId, managementItemId);
   }
 
-  private MenuItem findMenuItem(OrganizationId organizationId, MenuId mwpMenuId, MenuItemId mwpItemId) {
+  private MenuItem findMenuItem(OrganizationId organizationId, MenuId managementMenuId, MenuItemId managementItemId) {
     ApiResponse<List<fi.otavanopisto.mwp.client.model.Menuitem>> response =  
-        mwpApi.getApi(organizationId).kuntaApiMenusMenuIdItemsGet(mwpMenuId.getId());
+        managementApi.getApi(organizationId).kuntaApiMenusMenuIdItemsGet(managementMenuId.getId());
     
     if (!response.isOk()) {
       logger.severe(String.format("Menu finding failed on [%d] %s", response.getStatus(), response.getMessage()));
     } else {
-      for (fi.otavanopisto.mwp.client.model.Menuitem mwpItem : response.getResponse()) {
-        if (mwpItemId.getId().equals(String.valueOf(mwpItem.getId()))) {
-          return translateMenuItem(mwpItem);
+      for (fi.otavanopisto.mwp.client.model.Menuitem managementMenuItem : response.getResponse()) {
+        if (managementItemId.getId().equals(String.valueOf(managementMenuItem.getId()))) {
+          return translateMenuItem(managementMenuItem);
         }
       }
     }
     
     return null;
   }
   
-  private List<Menu> translateMenus(List<fi.otavanopisto.mwp.client.model.Menu> mwpMenus) {
+  private List<Menu> translateMenus(List<fi.otavanopisto.mwp.client.model.Menu> managementMenus) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "translateMenus" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)